.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+require: rubocop-rails
+
+Rails/FindBy:
+  Exclude:
+    - app/helpers/application_helper.rb
+
+Style/Documentation:
+  Enabled: false
```

Comment:
Will need to check to see how we should go about avoiding these errors in the future or if we can use this solution